client/script/ssCreate.js

```diff
@@ -423,9 +434,9 @@ var submissionHandling = function (inputSources, targetUrl) {
             // $verifyBox.prop('checked', false);
           }
           else {
-            alert(errMsg);
+            alertify.alert(errMsg);
```

Comment:
What happens here to the styling of the alert? If this is an error message, why not use the error image that's used elsewhere?